src/service/validation-service.js

```diff
@@ -35,21 +35,18 @@ class ValidationService {
         this.logger.info(`Validating assertionId: ${assertionId}`);
 
         this.validateAssertionId(assertion, assertionId);
-
-        // TODO: get assertion data in one call
-        await this.validateAssertionSize(blockchain, assertionId, assertion);
-        await this.validateTriplesNumber(blockchain, assertionId, assertion);
-        await this.validateChunkSize(blockchain, assertionId, assertion);
-
-        this.logger.info(`Assertion integrity validated!`);
-    }
-
-    async validateAssertionSize(blockchain, assertionId, assertion) {
-        const blockchainAssertionSize = await this.blockchainModuleManager.getAssertionSize(
+        const blockchainData = await this.blockchainModuleManager.getAssertionData(
             blockchain,
             assertionId,
         );
+        await this.validateAssertionSize(blockchainData.size, assertion);
+        await this.validateTriplesNumber(blockchainData.triplesNumber, assertion);
+        await this.validateChunkSize(blockchainData.chunksNumber, assertion);
 
+        this.logger.info(`Assertion integrity validated!`);
```

Comment:
I know that we had it before, but this log is not informative at all, has no connection with any assertionId or anything